### ISSUE TITLE
useless error message for binary source methods setting

### ISSUE THREAD

**Issue by rpoleski** [rpoleski/MulensModel#132]
A recent finding from my student:

```python
model = mm.Model({'t_0_1': 10, 'u_0_1': 0.2, 't_0_2': 30, 'u_0_2': 0.4, 't_E': 50, 'rho_1': 0.6})
model.set_magnification_methods([9, "finite_source_uniform_Gould94", 11])
model.get_magnification(10., separate=True)
```

produces error message:
```
...
File "MulensModel/source/MulensModel/pointlens.py", line 211, in _get_point_lens_finite_source_magnification
    z = u / rho
TypeError: unsupported operand type(s) for /: 'float' and 'NoneType'
```
which is not very useful. It also suggests that there should be more tests of input in `Model.set_magnification_methods()`.

**Comment by rpoleski** [comment]
A very similar error message is produced by:

```python
model = mm.Model({'t_0': 10, 'u_0': 0.2, 't_E': 50})
model.set_magnification_methods([9, "finite_source_uniform_Gould94", 11])
model.get_magnification(10.)
```

**Comment by rpoleski** [comment]
The second issue was solved in [a48c647](https://github.com/rpoleski/MulensModel/commit/a48c647d3e34cab3d0bcdfd33817ba0301f1993d).
